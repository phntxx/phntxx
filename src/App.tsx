import React from "react";
import styled from "@emotion/styled";
import { Global, css } from "@emotion/react";

import Heading from "./components/heading";
import About from "./components/about";
import Contact from "./components/contact";

const Container = styled.div`
  padding: 1rem;

  @media (min-width: 766px) {
    max-width: 30%;
  }
`;

const App = () => (
  <>
    <Global
      styles={css`
        body {
          font-family: "Work Sans", sans-serif;
        }

        #root {
          display: flex;
          align-items: center;
          justify-content: center;
        }

        html,
        body,
        #root {
          height: 100%;
          width: 100%;
          margin: 0;
        }
      `}
    />
    <Container>
      <Heading />
      <About />
      <Contact />
    </Container>
  </>
);

export default App;
