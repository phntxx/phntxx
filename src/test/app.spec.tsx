import { render } from "@testing-library/react";
import App from "../app";

describe("app.tsx", () => {
  it("should render component without label without crashing", () => {
    const { asFragment } = render(<App />);
    expect(asFragment).toMatchSnapshot();
  });
});
