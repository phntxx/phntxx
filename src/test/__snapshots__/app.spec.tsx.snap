// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app.tsx should render component without label without crashing 1`] = `[Function]`;
